pkg/minikube/reason/known_issues.go

```diff
@@ -578,6 +578,16 @@ Alternatively, you can try upgrading to the latest hyperkit version, or using an
 		},
 		Regexp: re(`VBoxManage not found. Make sure VirtualBox is installed and VBoxManage is in the path`),
 	},
+
+	// QEMU
+	{
+		Kind: Kind{
```

Comment:
is it possible to detect it before it reaches to regex ? if not that is still  valuable

Reply:
That's a good question, I'll look into that